tests/test_mcp_toolset.py

```diff
@@ -99,3 +105,257 @@ def test_mcpserver_no_headers():
     )
 
     assert mcp_toolset1.get_headers() is None
+
+
+def test_streamable_http_list_authorizations():
```

Comment:
A. Let start and reorginize our tests better. Seperate the sse tests from http test.
B. Change tests name to describe what they are checking and not what they are doing. e.g. `test_streamable_http_list_authorizations` -> describe that the test call list_authrizations but not what it checks. consider use given__when__then pattern.